src/components/pre-search/custom-query-preview.vue

```diff
@@ -7,53 +7,30 @@
       :queryFeature="queryFeature"
       #default="{ results, totalResults }"
     >
-      <div
-        class="x-list"
-        :class="
-          $x.device === 'mobile'
-            ? 'x-list--gap-01 x-margin--bottom-08'
-            : 'x-list--gap-05 x-margin--bottom-11'
-        "
-      >
-        <div :class="{ 'x-padding--left-05 x-padding--right-05': $x.device === 'mobile' }">
-          <h1 class="x-title1" :class="{ 'x-title1-sm': $x.device === 'mobile' }">{{ title }}</h1>
-          <div class="x-list x-list--horizontal x-list--align-center">
-            <span class="x-list__item--expand x-title3">
-              {{ query }}
-              <span class="x-text1">({{ totalResults }})</span>
-            </span>
+      <div class="x-mb-40 x-flex x-flex-col x-gap-2 desktop:x-mb-64 desktop:x-gap-16">
+        <div class="max-desktop:x-px-16">
+          <h1 class="max-desktop:x-title1-sm desktop:x-title1">
+            {{ title }}
+          </h1>
+        </div>
+
+        <CustomSlidingPanel>
+          <template #header>
             <BaseEventButton
               :events="getEvent(query)"
-              class="x-button x-button-lead x-button-ghost"
-              :class="{ 'x-uppercase': $x.device === 'mobile' }"
+              class="x-button x-button-tight max-desktop:x-px-16"
             >
-              {{ $t('queryPreview.viewResults') }}
-              <ChevronRightIcon />
+              {{ query }}
+              ({{ totalResults }})
+              <ArrowRightIcon class="x-icon-lg" />
             </BaseEventButton>
-          </div>
-        </div>
-        <SlidingPanel
-          :showButtons="$x.device !== 'mobile'"
-          :resetOnContentChange="false"
-          buttonClass="x-button-lead x-button-circle x-button-ghost x-padding--00"
-          scrollContainerClass="desktop:x-sliding-panel-fade-sm"
-          class="x-sliding-panel-show-buttons-on-hover"
-        >
-          <template #sliding-panel-left-button>
-            <ChevronLeftIcon class="x-icon-lg" />
           </template>
-          <ul
-            class="x-list x-list--horizontal x-list--gap-05"
-            :class="{ 'x-padding--left-05 x-padding--right-05': $x.device === 'mobile' }"
-          >
-            <li v-for="result in results" :key="result.id">
-              <Result :result="result" class="x-query-preview__item" />
-            </li>
-          </ul>
-          <template #sliding-panel-right-button>
-            <ChevronRightIcon class="x-icon-lg" />
-          </template>
-        </SlidingPanel>
+          <ItemsList :items="results" class="x-flex x-gap-16 x-pt-16 max-desktop:x-px-16">
```

Comment:
```suggestion
          <ItemsList :items="results" class="x-flex x-gap-16 x-pt-4 max-desktop:x-pt-0 max-desktop:x-px-16">
```
Distance between `BaseEventButton` it's less in designs, 16px & 10px mobile. As the button has its own white space inside, I think the padding defined should be a bit less.